### PR TITLE
Update Cluster Autoscaler version to 1.19.0

### PR DESCRIPTION
**Pull request by vivekbagade**
/kind bug
/priority critical-urgent
/sig autoscaling
/assign @mwielgus

```release-note
Updated Cluster Autoscaler to 1.19.0;
```

### CODE REVIEW

**Comment by MaciekPytel** [comment]
/lgtm
/approve
We also need to cherry-pick this to release-1.19 branch.

File: cluster/gce/manifests/cluster-autoscaler.manifest [hunk 1]
```diff
@@ -17,7 +17,7 @@
         "containers": [
             {
                 "name": "cluster-autoscaler",
-                "image": "k8s.gcr.io/cluster-autoscaler:v1.18.0",
+                "image": "k8s.gcr.io/cluster-autoscaler:v1.19.0",
```
Comment (gjtempleton) [inline cluster/gce/manifests/cluster-autoscaler.manifest:20]: Does this not live under `k8s.gcr.io/autoscaling/cluster-autoscaler:v1.19.0` now?

Comment (MaciekPytel) [reply cluster/gce/manifests/cluster-autoscaler.manifest]: Good catch, I missed that initially.

**Comment by vivekbagade** [comment]
@MrHohn PTAL

**Comment by MrHohn** [comment]
@vivekbagade Looks like this has been LGTMed and approved. And unfortunately I don't have the privilege to set milestone either.

**Pull request merged**